hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsScheduler.java

```diff
@@ -214,7 +220,9 @@ public void resourceOffers(SchedulerDriver driver, List<Offer> offers) {
       journalNodesResolvable = dnsResolver.journalNodesResolvable();
     }
     for (Offer offer : offers) {
```

Comment:
to do our best to keep the style as consistent as possible, let's make this line hdfsMesosConstraints.constraintsAllow(offer) instead and remove this